### PR TITLE
feat: add templ support

### PR DESCRIPTION
**Pull request by zbindenren**

### CODE REVIEW

File: lua/conform/formatters/templ.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://templ.guide",
```
Comment (stevearc) [inline lua/conform/formatters/templ.lua:4]: Could you link to the fmt command directly? https://templ.guide/commands-and-tools/cli/#formatting-templ-files

**Review by stevearc — CHANGES_REQUESTED** [review]
Just one nit

File: lua/conform/formatters/templ.lua (hunk 1 above)
Comment (zbindenren) [reply lua/conform/formatters/templ.lua]: Updated the URL in Readme and lua file.

**Comment by stevearc** [comment]
Great, thanks for the PR!

**Pull request merged**